### ISSUE TITLE
retrieval results

### ISSUE THREAD

**Issue by young-yangb** [HumaticsLAB/SEAM-Match-RCNN#2]
Hi, the retrieval results I reproduced seems inaccurate and I have no idea what the problem is.
I used the weights pre-trained on DeepFashion2, with 2  gpus for distributed training, and the evaluation result is as follows:

Top-1 Retrieval Accuracy: 0.0626
Top-5 Retrieval Accuracy: 0.1818
Top-10 Retrieval Accuracy: 0.2727
Top-20 Retrieval Accuracy: 0.3667
**************************************************
Top-1 Retrieval Accuracy Product Max: 0.2727
Top-5 Retrieval Accuracy Product Max: 0.5253
Top-10 Retrieval Accuracy Product Max: 0.6263
Top-20 Retrieval Accuracy Product Max: 0.7778
**************************************************
Top-1 Retrieval Accuracy Product Avg Desc: 0.1212
Top-5 Retrieval Accuracy Product Avg Desc: 0.3535
Top-10 Retrieval Accuracy Product Avg Desc: 0.4545
Top-20 Retrieval Accuracy Product Avg Desc: 0.5455
**************************************************
Top-1 Retrieval Accuracy Product Aggr Desc: 0.1818
Top-5 Retrieval Accuracy Product Aggr Desc: 0.4141
Top-10 Retrieval Accuracy Product Aggr Desc: 0.4646
Top-20 Retrieval Accuracy Product Aggr Desc: 0.5455
**************************************************
Top-1 Retrieval Accuracy Product Avg Dist: 0.1010
Top-5 Retrieval Accuracy Product Avg Dist: 0.3434
Top-10 Retrieval Accuracy Product Avg Dist: 0.4242
Top-20 Retrieval Accuracy Product Avg Dist: 0.5556
**************************************************
Top-1 Retrieval Accuracy Product Max Dist: 0.0606
Top-5 Retrieval Accuracy Product Max Dist: 0.2323
Top-10 Retrieval Accuracy Product Max Dist: 0.4444
Top-20 Retrieval Accuracy Product Max Dist: 0.5051
**************************************************
Top-1 Retrieval Accuracy Product Max Score: 0.1111
Top-5 Retrieval Accuracy Product Max Score: 0.2626
Top-10 Retrieval Accuracy Product Max Score: 0.3939
Top-20 Retrieval Accuracy Product Max Score: 0.5051
**************************************************

 Regular ONLY
Top-1 Retrieval Accuracy: 0.0826
Top-5 Retrieval Accuracy: 0.2261
Top-10 Retrieval Accuracy: 0.3232
Top-20 Retrieval Accuracy: 0.4217
**************************************************
Top-1 Retrieval Accuracy Product Avg Desc: 0.1739
Top-5 Retrieval Accuracy Product Avg Desc: 0.4493
Top-10 Retrieval Accuracy Product Avg Desc: 0.5362
Top-20 Retrieval Accuracy Product Avg Desc: 0.6377
**************************************************
Top-1 Retrieval Accuracy Product Aggr Desc: 0.2174
Top-5 Retrieval Accuracy Product Aggr Desc: 0.5217
Top-10 Retrieval Accuracy Product Aggr Desc: 0.5652
Top-20 Retrieval Accuracy Product Aggr Desc: 0.6522
**************************************************
Top-1 Retrieval Accuracy Product Avg Dist: 0.1449
Top-5 Retrieval Accuracy Product Avg Dist: 0.4493
Top-10 Retrieval Accuracy Product Avg Dist: 0.5072
Top-20 Retrieval Accuracy Product Avg Dist: 0.6667
**************************************************
Top-1 Retrieval Accuracy Product Max Dist: 0.0870
Top-5 Retrieval Accuracy Product Max Dist: 0.2899
Top-10 Retrieval Accuracy Product Max Dist: 0.5362
Top-20 Retrieval Accuracy Product Max Dist: 0.6087
**************************************************
Top-1 Retrieval Accuracy Product Max Score: 0.1449
Top-5 Retrieval Accuracy Product Max Score: 0.3333
Top-10 Retrieval Accuracy Product Max Score: 0.4783
Top-20 Retrieval Accuracy Product Max Score: 0.5942
**************************************************

 Hard ONLY
Top-1 Retrieval Accuracy: 0.0167
Top-5 Retrieval Accuracy: 0.0800
Top-10 Retrieval Accuracy: 0.1567
Top-20 Retrieval Accuracy: 0.2400
**************************************************
Top-1 Retrieval Accuracy Product Avg Desc: 0.0000
Top-5 Retrieval Accuracy Product Avg Desc: 0.1333
Top-10 Retrieval Accuracy Product Avg Desc: 0.2667
Top-20 Retrieval Accuracy Product Avg Desc: 0.3333
**************************************************
Top-1 Retrieval Accuracy Product Aggr Desc: 0.1000
Top-5 Retrieval Accuracy Product Aggr Desc: 0.1667
Top-10 Retrieval Accuracy Product Aggr Desc: 0.2333
Top-20 Retrieval Accuracy Product Aggr Desc: 0.3000
**************************************************
Top-1 Retrieval Accuracy Product Avg Dist: 0.0000
Top-5 Retrieval Accuracy Product Avg Dist: 0.1000
Top-10 Retrieval Accuracy Product Avg Dist: 0.2333
Top-20 Retrieval Accuracy Product Avg Dist: 0.3000
**************************************************
Top-1 Retrieval Accuracy Product Max Dist: 0.0000
Top-5 Retrieval Accuracy Product Max Dist: 0.1000
Top-10 Retrieval Accuracy Product Max Dist: 0.2333
Top-20 Retrieval Accuracy Product Max Dist: 0.2667
**************************************************
Top-1 Retrieval Accuracy Product Max Score: 0.0333
Top-5 Retrieval Accuracy Product Max Score: 0.1000
Top-10 Retrieval Accuracy Product Max Score: 0.2000
Top-20 Retrieval Accuracy Product Max Score: 0.3000
**************************************************
Rank median: 36.0; rank 1st quartile: 7.0; rank 3rd quartile: 140.25
Average Track Length: 9.292929292929292

**Comment by joppichristian** [comment]
Our results are obtained on a single GPU, training over 60 epochs. Try to run a training on a single GPU for 75 epochs, with a single learning rate step at 30 epochs. Unfortunately, results changes due to the random variable in selecting frames, but not so much.
